packages/terra-consumer-nav/src/components/nav-help/NavHelpContent.jsx

```diff
@@ -12,35 +12,46 @@ const cx = classNames.bind(styles);
 class NavHelpContent extends React.Component {
   constructor() {
     super();
-    this.state = ({ isOpen: false });
+    this.state = ({ openTogglers: [] });
 
     this.handleToggle = this.handleToggle.bind(this);
   }
 
-  handleToggle() {
-    this.setState({ isOpen: !this.state.isOpen });
+  handleToggle(toggleIndex) {
```

Comment:
seems like this logic would all be alot simpler if you just had a hash, where the key was the toggleIndex, and the value was a boolean for isOpen. then handleToggle would be
```
state = {
  togglers: {
     1: true,
     2: false,
     3: false,
  }
}
handleToggle(toggleIndex){
   this.setState({
      togglers: {...this.state.togglers, toggleIndex: !this.state.togglers[toggleIndex]}
    })
}
```

Reply:
you're right, hash is easier here, will update this